apps/forklift/test/unit/forklift/data_writer_test.exs

```diff
@@ -249,12 +232,15 @@ defmodule Forklift.DataWriterTest do
         technical: %{systemName: "some_system_name"}
```

Comment:
Rename this test to "Compaction *is* kicked off if end_of_data message is received".

Remove ingestion_status field.